x/dispensation/utils/parser_test.go

```diff
@@ -59,20 +61,84 @@ func init() {
 	SetConfig()
 }
 func TestParseInput(t *testing.T) {
-	filename := "input.json"
-	createInput(t, filename)
-	defer removeFile(t, filename)
-	inputs, err := utils.ParseInput(filename)
+	file := "input.json"
+	createInput(t, file)
+	defer removeFile(t, file)
+	inputs, err := utils.ParseInput(file)
 	assert.NoError(t, err)
 	assert.Equal(t, len(inputs), 2)
 }
 
 func TestParseOutput(t *testing.T) {
-	filename := "output.json"
+	file := "output.json"
 	count := 3000
-	createOutput(filename, count)
-	defer removeFile(t, filename)
-	outputs, err := utils.ParseOutput(filename)
+	createOutput(file, count)
+	defer removeFile(t, file)
+	outputs, err := utils.ParseOutput(file)
 	assert.NoError(t, err)
 	assert.Equal(t, len(outputs), count)
 }
+
+// TODO Add the following utils as its own separate cmd
+
+func TestAddressFilter(t *testing.T) {
+	var addresStrings []string
+	file, err := filepath.Abs("sample.json")
+	if err != nil {
+		panic("Err getting filepath :" + err.Error())
+	}
+	o, err := ioutil.ReadFile(file)
+	if err != nil {
+		panic("Err Reading file :" + err.Error())
+	}
+	err = json.Unmarshal(o, &addresStrings)
+	if err != nil {
+		panic("Err Unmarshall :" + err.Error())
+	}
+	for _, add := range addresStrings {
+		_, err := sdk.AccAddressFromBech32(add)
+		if err != nil {
+			fmt.Println("Invalid :", add)
```

Comment:
Should we panic here as well?

Reply:
this test is just to print out the invalid address . So that they can be removed from the input list .

Reply:
Its actaully a utility and not a test , jsut kept it under tests temporarily